<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssignColorsParameters</class>
 <widget class="QWidget" name="AssignColorsParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assign Colors</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>apply colors:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="ApplyDefaultColorBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Click here to assign this color to all the associated entities.</string>
     </property>
     <property name="text">
      <string>color</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="ApplyDefaultPaletteBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Click here to assign a color from this palette to each associated entity.</string>
     </property>
     <property name="text">
      <string>palette</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="RemoveColorBtn">
     <property name="toolTip">
      <string>Click here to reset the entity so it inherits the default color.</string>
     </property>
     <property name="text">
      <string>inherited default</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="ChooseColorBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Click here to change the default color above.</string>
     </property>
     <property name="text">
      <string>change color</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="ChoosePaletteBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Click here to change the default palette above.</string>
     </property>
     <property name="text">
      <string>change palette</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
